scripts/mumble-version.py

```diff
@@ -122,8 +123,8 @@ def main():
 
     if len(latestCommitTags) == 1:
         # Most recent commit is tagged -> this is a tagged release version
-        # Use the tag as the version-string
-        version = latestCommitTags[0]
+        # Extract suffix from tag name, assuming it begins with the version
+        suffix = latestCommitTags[0][5:]
```

Comment:
This assumes a `x.x.x` format of the version. It will break if we reached something like `1.4.10`

We probably have to use a RegEx like `\d+\.\d+\.\d+` in order to match the prefixing version and then figure out its length from the match

Reply:
Or just drop the tag logic entirely, as you proposed.